lib/rpmsg/rpmsg.c

```diff
@@ -192,20 +192,18 @@ static void rpmsg_unregister_endpoint(struct rpmsg_endpoint *ept)
 	metal_list_del(&ept->node);
 }
 
-int rpmsg_register_endpoint(struct rpmsg_device *rdev,
-			    struct rpmsg_endpoint *ept)
+void rpmsg_register_endpoint(struct rpmsg_device *rdev,
+			     struct rpmsg_endpoint *ept)
 {
 	ept->rdev = rdev;
-
 	metal_list_add_tail(&rdev->endpoints, &ept->node);
-	return RPMSG_SUCCESS;
 }
 
 int rpmsg_create_ept(struct rpmsg_endpoint *ept, struct rpmsg_device *rdev,
 		     const char *name, uint32_t src, uint32_t dest,
 		     rpmsg_ept_cb cb, rpmsg_ns_unbind_cb unbind_cb)
 {
-	int status;
+	int status = 0;
 	uint32_t addr = src;
```

Comment:
seems that the initialization is not mandatory but if you keep it please use RPMSG_SUCCESS

Reply:
The assignment is required otherwise it will be random value if src equals RPMSG_ADDR_ANY and dst doesn't equals RPMSG_ADDR_ANY.
Ok, I will change 0 to RPMSG_SUCCESS.